cmd/cut-release/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/plaid/go-envvar/envvar"
+)
+
+type envVars struct {
+	// Version is the new release version to use
+	Version string `envvar:"VERSION" default:"development"`
+}
+
+func main() {
+	env := envVars{}
+	if err := envvar.Parse(&env); err != nil {
+		log.Fatal(err)
+	}
+
+	updateHardCodedVersions(env.Version)
+
+	generateTypescriptClientDocs()
+}
+
+func generateTypescriptClientDocs() {
+	// Run `yarn install` to make sure `TypeDoc` dep is installed
+	cmd := exec.Command("yarn", "install")
+	cmd.Dir = "rpc/clients/typescript"
+	if err := cmd.Run(); err != nil {
+		log.Fatal(err)
```

Comment:
`err` here doesn't include the output from the command itself. I think we should also log [`cmd.CombinedOutput`](https://golang.org/pkg/os/exec/#Cmd.CombinedOutput).

Reply:
Edit: I just realized `CombinedOutput` actually runs the command so what we really want to do is replace `Run` with `CombinedOutput`.